python/tvm/relay/backend/contrib/uma/_template/backend.py

```diff
@@ -0,0 +1,53 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
```

Comment:
Given the out-of-tree usecase of UMA, I personally feel this might be better suited as an apps/ example, where we showcase to a "user" how UMA could be used. WDYT ? 

Even if its a template, then it would be still good demonstrate how to use that template in a "user" flow.

*user here means a out-of-tree dev that wants to attach accelerator codegen to TVM

Reply:
We will move it to apps

Reply:
done